packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -290,23 +266,45 @@ describeNoCompat("GC Random tests", (getTestObjectProvider) => {
         unreferenceRandomDataStore,
     ];
     const testTime = 60 * 1000; // 1 minute
+    let sleptTime = 0;
+    const plannedActions: { action: (() => Promise<void>); sleepTime: number; }[] = [];
+    while (sleptTime < testTime) {
+        const sleepTime: number = randomInteger(0, 100);
+        const action = random.pick(actionsList);
+        plannedActions.push({ action, sleepTime });
+        sleptTime += sleepTime;
+    }
 
-    it.skip(`Create and reference and unreference datastores with multiple containers. Seed: ${seed}`, async () => {
+    let plannedActionsString: string = "";
+    for (const action of plannedActions) {
+        plannedActionsString += `action: ${action.action.name}, sleepTime: ${action.sleepTime}\n`;
+    }
+
+    it.skip(`Planned Actions:\n${plannedActionsString}\nSeed: ${seed}`, async () => {
```

Comment:
>.skip [](http://example.com/codeflow?start=6&length=5)

why are we skipping ? Add comment ?

Reply:
The test takes a minute (for now, may potentially take longer) to run, it would increase CI times. Currently I am running this locally.

Reply:
Made it so that there was no skip and we can only manually run the test.